src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -659,33 +671,46 @@ const WorkflowView = _.flow(
             div([
               h(RadioButton, {
                 disabled: !!Utils.editWorkspaceError(ws) || currentSnapRedacted,
-                text: `Process multiple workflows from:`,
+                text: `Run workflow(s) with inputs defined by data table`,
                 name: 'process-workflows',
                 checked: this.isMultiple(),
                 onChange: () => this.selectMultiple(),
                 labelStyle: { marginLeft: '0.5rem' }
-              }),
-              h(Select, {
-                'aria-label': 'Entity type selector',
-                isClearable: false, isDisabled: currentSnapRedacted || this.isSingle() || !!Utils.editWorkspaceError(ws), isSearchable: false,
-                placeholder: 'Select data type...',
-                styles: { container: old => ({ ...old, display: 'inline-block', width: 200, marginLeft: '0.5rem' }) },
-                getOptionLabel: ({ value }) => Utils.normalizeLabel(value),
-                value: selectedEntityType,
-                onChange: selection => {
-                  const value = this.updateEntityType(selection)
-                  this.setState({ entitySelectionModel: this.resetSelectionModel(value) })
-                },
-                options: _.keys(entityMetadata)
-              }),
-              h(Link, {
-                disabled: currentSnapRedacted || this.isSingle() || !rootEntityType || !entityMetadata[rootEntityType] || !!Utils.editWorkspaceError(ws),
-                tooltip: Utils.editWorkspaceError(ws),
-                onClick: () => this.setState({ selectingData: true }),
-                style: { marginLeft: '1rem' }
-              }, ['Select Data'])
+              })
             ]),
-            div({ style: { marginLeft: '2rem', height: '1.5rem' } }, [`${this.describeSelectionModel()}`])
+            // !!this.isMultiple() && div({ style: { columns: 'auto', divider: true, ...styles.description, margin: '0.5rem 0 0 2rem' } }, [
+            !!this.isMultiple() && div({ style: { display: 'flex', margin: '0.5rem 0 0 2rem' } }, [
```

Comment:
It's already a boolean.
```suggestion
            this.isMultiple() && div({ style: { display: 'flex', margin: '0.5rem 0 0 2rem' } }, [
```